### PR TITLE
Multiplicative inverse

### PR DESCRIPTION
**Pull request by retabak**
whew, formatting and weschemifying ate up too much time!

### CODE REVIEW

File: lessons/Early-Math/multiplicative-inverse/langs/en-us/index.adoc [hunk 1]
```diff
@@ -40,28 +40,35 @@ Students discover the Multiplicative Inverse Property, which tells us that multi
...
 === Launch
 
-We know that the Commutative and Associative Properties apply to multiplication...but not division. But what if there was a way to _rewrite_ division as multiplication? Then we could apply the Commutative and Associative Properties to division expressions, too! Let’s explore this idea.
+We know that the Commutative and Associative Properties apply to multiplication... but not division. But what if there was a way to _rewrite_ division as multiplication? Then we could apply the Commutative and Associative Properties to division expressions, too! Let’s explore this idea.
 
-But first, let's play a game. Draw a two-column table on the board to record response. Label the left-hand side "Starting Value," and label the right-hand side "?" (see example below).
+But first, let's play a game. I will draw a two-column table on the board.I will label the left-hand side "Starting Value," and I will label the right-hand side "?"
 
+
+@ifnotslide{
```
Comment (schanzer) [inline lessons/Early-Math/multiplicative-inverse/langs/en-us/index.adoc:48]: Why `ifnotslide`? Use the `@table{}` directive to have this work for both lesson plans and slides!

File: lessons/Early-Math/multiplicative-inverse/langs/en-us/index.adoc [hunk 2]
```diff
@@ -40,28 +40,35 @@ Students discover the Multiplicative Inverse Property, which tells us that multi
 
 === Launch
 
-We know that the Commutative and Associative Properties apply to multiplication...but not division. But what if there was a way to _rewrite_ division as multiplication? Then we could apply the Commutative and Associative Properties to division expressions, too! Let’s explore this idea.
+We know that the Commutative and Associative Properties apply to multiplication... but not division. But what if there was a way to _rewrite_ division as multiplication? Then we could apply the Commutative and Associative Properties to division expressions, too! Let’s explore this idea.
 
-But first, let's play a game. Draw a two-column table on the board to record response. Label the left-hand side "Starting Value," and label the right-hand side "?" (see example below).
+But first, let's play a game. I will draw a two-column table on the board.I will label the left-hand side "Starting Value," and I will label the right-hand side "?"
```
Comment (schanzer) [inline lessons/Early-Math/multiplicative-inverse/langs/en-us/index.adoc]: I don't think the "?" column header is helpful, and it's slightly confused.

I'm wondering if you shouldn't start by telling students what a reciprocal of a number is: "the number that, when multiplied by the original, produces 1". Then you could title the second column "reciprocal".

File: lessons/Early-Math/multiplicative-inverse/langs/en-us/index.adoc [hunk 3]
```diff
@@ -96,25 +103,28 @@ Now that we understand what a reciprocal is, we are ready to think about how we
...
+
+@slidebreak
+
+Hopefully, you discovered two big ideas:
 
 - Dividing by @math{x} produces the same result as multiplying by its reciprocal, @math{\frac{1}{x}}.
 - Dividing by @math{\frac{1}{x}} produces the same result as multiplying by its reciprocal, @math{x}.
 
-In other words, when students see multiplication or division by a unit fraction (a fraction with a numerator of @math{1}), there is likely a path forward using mental computation only.
+In other words, when you see multiplication or division by a unit fraction (a fraction with a numerator of @math{1}), there is likely a path forward using *mental computation only*.
```
Comment (schanzer) [inline lessons/Early-Math/multiplicative-inverse/langs/en-us/index.adoc:117]: SOMEWHERE in here I was expecting to see the word "equivalent" or "equivalence". That's the overriding theme of the course, and it definitely applies here. Is there a way to work that word in, before students see it in the following section?

**Review by schanzer — CHANGES_REQUESTED** [review]
Overall this is SUPER close. It just needs a little polishing.

I took a pass over the language, and sanded down some of the more complex vocabulary and grammar. Feel free to push back if you don't like some of my changes!

File: lessons/Early-Math/multiplicative-inverse/langs/en-us/index.adoc (hunk 1 above)
Comment (retabak) [reply lessons/Early-Math/multiplicative-inverse/langs/en-us/index.adoc:48]: I did not know that!

**Comment by retabak** [comment]
@schanzer - I believe I have addressed all feedback! LMK if you'd like to review again or if I should squash + merge.

**Review by schanzer — APPROVED** [review]
Looks great! Squash and merge away!

**Pull request merged**